malaria24/ona/tasks.py

```diff
@@ -17,6 +17,10 @@
 from onapie.client import Client
 
 from go_http.send import HttpApiSender
+from requests.auth import HTTPBasicAuth
+
+USER_NAME = 'fake@example.com'
```

Comment:
I think this should probably move to the settings file where we can try and get them from environment variables, look at the same thing in `ndoh-hub`:
https://github.com/praekeltfoundation/ndoh-hub/blob/develop/ndoh_hub/settings.py#L273

Reply:
Thanks for giving me direction on this. It will be implemented